tb_db/parsers.py

```diff
@@ -216,51 +216,49 @@ def parse_run_ids(locations_path):
 
 
 # libraries
-def parse_libraries(qc_path, locations_path):
+def parse_libraries(qc_path, sequencing_run_id):
     qcs = []
     with open(qc_path, 'r') as f:
         reader = csv.DictReader(f)
         for row in reader:
             qc = {
                 'sample_id': row['sample_id'],
-                'sample_name': row['sample_id'],
+                #'sample_name': row['sample_id'],
+                'sequencing_run_id' : sequencing_run_id,
                 'most_abundant_species_name':row['most_abundant_species_name'],
                 'most_abundant_species_fraction_total_reads': float(row['most_abundant_species_fraction_total_reads']),
```

Comment:
This is optional but I have a suggestion:

I try to wrap conversions to `int` and `float` in `try` / `catch` blocks like this:

```
try:
    total_bases = int(row['total_bases'])
except ValueError as e:
    total_bases = None
```

If there are several fields that need to be converted then I'll do something like this:

```
float_fields = [
    'average_base_quality',
    'percent_bases_above_q30',
    'percent_gc',
]
for field in float_fields:
    try:
        row[field] = float(row[field])
    except ValueError as e:
        row[field] = None
```